src/SourceBuild/content/repo-projects/sdk.proj

```diff
@@ -13,18 +13,19 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <RepositoryReference Include="arcade" />
```

Comment:
Can you help me understand why arcade is removed from some repos but not others?  What are the rules/guidelines?

Reply:
Does this make the graph harder to maintain overtime when repos and dependencies are added/removed versus basing it on direct dependencies?

Reply:
FWIW we prefer direct dependencies over transitive dependencies for PackageReference and ProjectReferences. It makes the definition more explicit. Another thought: If a primary dependency is only referenced transitively, what would happen if the dependency that brings the transitive dependency in, stops depending on that dependency?

Reply:
Yeah, I would prefer extra edges for top level dependencies. I understand cases where if I depend on X, then I depend on what X depends on and so I should get those transitively for free. If I remove my dependency on X, then I should implicitly remove my dependencies on X's dependencies.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/608e3adab65aca27adcc828a17383c6b401466dc